offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/AppView.groovy

```diff
@@ -153,19 +158,23 @@ class AppView extends VerticalLayout {
 
         Button searchPersonBtn
 
+        Button createProjectBtn
+
         TomatoFeatures() {
             this.createOfferBtn = new Button("New Offer")
             this.createCustomerBtn = new Button("New Customer")
             this.createAffiliationBtn = new Button("New Affiliation")
             this.overviewBtn = new Button("Offer Overview")
             this.searchPersonBtn = new Button("Search Customer")
+            this.createProjectBtn = new Button("Create Project")
```

Comment:
If we want to keep the use case in the menu we need to offer the user the possibility to select an offer

Reply:
changes in this class should be reverted since we now use the overview view :)